components/org.wso2.carbon.micro.integrator.extensions/org.wso2.carbon.micro.integrator.management.apis/src/main/java/org/wso2/carbon/micro/integrator/management/apis/models/dataServices/DataServicesList.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.carbon.micro.integrator.management.apis.models.dataServices;
+
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.annotations.SerializedName;
+
+public class DataServicesList {
+    private int count;
```

Comment:
What is the purpose of `count`?

Reply:
```json
{
  "count": 1,
  "list": [
    {
      "name": "findDoctor",
      "wsdl1_1": "http://localhost:8280/services/findDoctor?wsdl",
      "wsdl2_0": "http://localhost:8280/services/findDoctor?wsdl2"
    }
  ]
}
```

`count` is corresponding to the `count` in the preceding response.

Reply:
I believe comprehensively representing all responses using Java objects is better in terms of readability and extendability, than to construct JSON/XML responses manually.